tools/c7n_azure/c7n_azure/query.py

```diff
@@ -79,6 +81,9 @@ def __init__(self, manager):
         self.manager = manager
         self.query = self.resource_query_factory(self.manager.session_factory)
 
+    def validate(self):
```

Comment:
Why is this validate method needed?

Reply:
https://github.com/cloud-custodian/cloud-custodian/pull/4924#discussion_r333024918
tldr; not all resources can be queried by resource-graph

Reply:
but shouldn't it be actually validating that then?